python/src/trezorlib/btc.py

```diff
@@ -416,7 +562,128 @@ def copy_tx_meta(tx: messages.TransactionType) -> messages.TransactionType:
+    preauthorized: bool = False,
+    unlock_path: Optional[List[int]] = None,
+    unlock_path_mac: Optional[bytes] = None,
+    **kwargs: Any,
+) -> Tuple[Sequence[Optional[bytes]], bytes]:
+    """Sign a Bitcoin-like transaction.
+
+    Returns a list of signatures (one for each provided input) and the
+    network-serialized transaction.
+
+    In addition to the required arguments, it is possible to specify additional
+    transaction properties (version, lock time, expiry...). Each additional argument
+    must correspond to a field in the `SignTx` data type. Note that some fields
+    (`inputs_count`, `outputs_count`, `coin_name`) will be inferred from the arguments
+    and cannot be overriden by kwargs.
+    """
+    anti_exfil_signatures, serialization = sign_tx_common(
+        client,
+        coin_name,
+        inputs,
+        outputs,
+        details,
+        prev_txes,
+        payment_reqs,
+        preauthorized,
+        unlock_path,
+        unlock_path_mac,
+        False,
+        None,
+        **kwargs,
+    )
+    return [
+        s.signature if s is not None else None for s in anti_exfil_signatures
+    ], serialization
+
+
+@session
+def sign_tx_new(
+    client: "TrezorClient",
+    coin_name: str,
+    inputs: Sequence[messages.TxInputType],
+    outputs: Sequence[messages.TxOutputType],
+    details: messages.SignTx | None = None,
+    prev_txes: TxCacheType | None = None,
+    payment_reqs: Sequence[messages.TxAckPaymentRequest] = (),
+    preauthorized: bool = False,
+    unlock_path: List[int] | None = None,
+    unlock_path_mac: bytes | None = None,
+    use_anti_exfil: bool = True,
+    entropy_list: List[bytes] | None = None,
+    **kwargs: Any,
+) -> Sequence[AntiExfilSignature | None]:
+    """Sign a Bitcoin-like transaction.
+
+    Returns a list of `AntiExfilSignature` objects (one for each provided input).
+
+    If `use_anti_exfil` is set to `True`, the anti-exfilitration protocol will be
+    used. The purpose of this protocol is to prevent the device from leaking
+    its secrets through the signatures. In this case, `AntiExfilSignature` objects
+    will have non-emtpy fields `entropy` and `nonce_commitment`. It's the caller
+    responsibility to verify the signature and the nonce commitment. The caller
+    can optionally provide a list of entropies to be used in the protocol. Ideally,
+    the caller should provide the same list of entropies if the signing is repeated
+    due to a error to prevent the device to perform nonce-grinding attacks.
+
+    In addition to the required arguments, it is possible to specify additional
+    transaction properties (version, lock time, expiry...). Each additional argument
+    must correspond to a field in the `SignTx` data type. Note that some fields
+    (`inputs_count`, `outputs_count`, `coin_name`) will be inferred from the arguments
+    and cannot be overriden by kwargs.
+    """
+    anti_exfil_signatures, serialization = sign_tx_common(
```

Comment:
The `serialization` variable is unused. I would either use it or replace it by underscore.

```suggestion
    anti_exfil_signatures, _ = sign_tx_common(
```